web/src/Root.tsx

```diff
@@ -501,6 +501,9 @@ const Root = () => {
             isEmbeddedClusterEnabled={Boolean(
               state.adminConsoleMetadata?.isEmbeddedCluster
             )}
+            isEmbeddedClusterNodeWaiting={
```

Comment:
why not keep it the same name?
```suggestion
            isEmbeddedClusterWaitingForNodes ={
```